### DISCUSSION TITLE
URL Monitoring

### DISCUSSION THREAD

**Discussion by kisanity** [signalfx/splunk-otel-collector-chart#569] [General] [answered]
Hello,

Is this the only way to monitor URLs: https://docs.splunk.com/Observability/gdi/http/http.html?

What if we have 100s of URLs that we need to monitor, can we point the config in the Helm values.yaml to a specific file? If so, how?

We're using the Helm chart on OpenShift.

Thanks

**Comment by atoulme** [comment, marked as answer]
This might be something to configure in the collector itself - please get in touch and open an idea. A new component to the collector recently added support for URL monitoring.

**Reply by atoulme** [reply]
See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/18823 for an idea and current progress on this work.
